pyproject.toml

```diff
@@ -59,8 +59,9 @@ tf = [
     "tf2onnx>=1.9.2",
 ]
 torch = [
-    "torch>=1.8.0",
-    "torchvision>=0.9.0",
+    "torch>=1.12.0",
+    "torchvision>=0.13.0",
+    "onnx>=1.12.0"
```

Comment:
I suggest not modifying lib version specifiers in this PR

Reply:
moved to other PR